### DISCUSSION TITLE
About the main output columns

### DISCUSSION THREAD

**Discussion by Sunmile** [vdemichev/DiaNN#525] [Q&A] [answered]
Hi,
Thanks for developing the powerful software DIA-NN. It really helps a lot to analyze the DIA data.
I noticed that some columns in the main output (report.tsv) are not mentioned and explained in the README. For example, the columns named: '**Evidence**' and '**Decoy.Evidence**'.
What's the reference of '**Evidence**' and '**Decoy.Evidence**'?
Since '**Mass.Evidence**' is the indicator of whether this row is a good identification of a peptide, does the '**Decoy.Evidence**' mean something like the probability that this identification is a decoy, and we should filter those rows with the high score of decoy.evidence out? If my understanding of '**Decoy.Evidence**' is correct, what's the meaning of some rows with both high scores of '**Mass.Evidence**' and '**Decoy.Evidence**'?

Many thanks,
Sunmile
<img width="420" alt="image" src="https://user-images.githubusercontent.com/28746628/194745574-0150e362-57fe-470c-bfbc-349be5187fef.png">

**Comment by vdemichev** [comment, marked as answer]
Hi Sunmile,

The columns that are not explained are there for development purposes, so should not be used.

Best,
Vadim

**Reply by Sunmile** [reply]
Dear Vadim,

Got it. Thanks for your prompt reply.

Best,
Sunmile
